src/app/pages/IdxPage/index.test.jsx

```diff
@@ -13,8 +16,12 @@ let container;
 
 describe('IdxPage', () => {
   beforeEach(async () => {
-    fetch.mockResponse(JSON.stringify(persianMostReadData));
-    fetch.mockResponse(JSON.stringify(dariRadioScheduleData));
+    fetchMock.restore();
+    fetchMock.mock(mostReadEndpoint, persianMostReadData);
+    fetchMock.mock(
+      '/data/persian/bbc_dari_radio/schedule.json',
```

Comment:
one last thing we can do is this move this out to a const above, similar to `mostReadEndpoint` 🙈

Reply:
Done now:)

Reply:
Thanks 💯 , happy for this to be merged once CI passes.